src/script.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable import/no-cycle */
 // Import the stylesheet
 import './style.css';
```

Comment:
- [ ] Same here,  Kindly remove all eslint disablement lines and try to fix eslint errors yourself.